src/components/timeline/Ruler.css

```diff
@@ -50,3 +50,14 @@
   right: 0;
   padding-right: 5px;
 }
+
+@media (forced-colors: active) {
+  /* The background gradient we use to display the notch are not visible in HCM.
+     We need to use a plain color, and we can't set the background color otherwise
+     it also covers the text */
```

Comment:
do you know why it also covers the text? It doesn't make sense to me, but maybe the forced colors makes things inherit styles when they otherwise shouldn't?

Reply:
added some comments